File: src/SmartBuildClient/src/app/models/customer.ts
import { Address } from './address';

export class Customer {
  constructor(public customerId?: number,
    public name?: string,
    public birthDate?: string,
    public rg?: string,
    public cpf?: string,
    public address?: Address,
    public homePhone?: string,
    public workPhone?: string,
    public referencePhone?: string,
    public cellPhone?: string,
    public email?: string,
    public isDeleted?: boolean) {
  }
}

File: src/SmartBuild.Entities/Customers/Customer.cs
﻿using System;

namespace SmartBuild.Entities.Customers
{
    public class Customer : IDeletable
    {
        public int CustomerId { get; set; }
        public string Name { get; set; }
        public DateTime? BirthDate { get; set; }
        public string RG { get; set; }
        public string CPF { get; set; }
        public Address Address { get; set; }
        public string HomePhone { get; set; }
        public string WorkPhone { get; set; }
        public string ReferencePhone { get; set; }
        public string CellPhone { get; set; }
        public string Email { get; set; }
        public bool IsDeleted { get; set; }
    }
}
